docker/Dockerfile

```diff
@@ -7,10 +7,14 @@ ARG GROUP=factorio
 ARG PUID=845
 ARG PGID=845
 
+# version checksum of the archive to download
+ARG VERSION
+ARG SHA256
+
 ENV PORT=34197 \
     RCON_PORT=27015 \
-    VERSION={{ version }} \
-    SHA1={{ sha1 }} \
+    VERSION=${VERSION:?} \
```

Comment:
The `:?` here will cause the build to fail if the value is not supplied.
It saves time by not waiting until the curl call happens.